src/utils/Logger.ts

```diff
@@ -53,10 +54,11 @@ class Logger {
   error = (
     tag: string,
     message: string,
-    error?: Error,
+    unknownError?: unknown,
```

Comment:
Are there still places where Logger is called without ensureError?

Reply:
Yes, over 100